BalancedBinaryTree.java

```diff
@@ -0,0 +1,41 @@
+package competitiveCoding4;
+
+
+//Time COmplexity: O(n^2) because nodes of trees are traversed twice.
```

Comment:
If nodes are traversed twice then complexity should be 2n.
Think about it again.